src/Umbraco.Web/Editors/MacroRenderingController.cs

```diff
@@ -156,7 +156,8 @@ public HttpResponseMessage CreatePartialViewMacroWithFile(CreatePartialViewMacro
             {
                 Alias = macroName.ToSafeAlias(),
                 Name = macroName,
-                MacroSource = model.VirtualPath.EnsureStartsWith("~")
+                MacroSource = model.VirtualPath.EnsureStartsWith("~"),
+                MacroType = MacroTypes.PartialView
```

Comment:
I guess we can also set this in the ctor of the Macro model - since we only have one macro type now :) looks good, i'll merge